mep/people/models.py

```diff
@@ -26,6 +26,17 @@
 
 logger = logging.getLogger(__name__)
 
+VIAF_SIGNAL_ACTIVE=True
+class ViafSignalHandler:
+    def disconnect():
+        global VIAF_SIGNAL_ACTIVE
+        VIAF_SIGNAL_ACTIVE = False
+    def connect():
+        global VIAF_SIGNAL_ACTIVE
+        VIAF_SIGNAL_ACTIVE = True
+    def is_connected():
+        return VIAF_SIGNAL_ACTIVE
+
```

Comment:
> Is that not an option here because of the many-to-many nationality relationship?
Yes, that's my sense from the docs and a quick test.

I'll look into alternatives re: global variable, etc. Right now the solution above is effectively a verbose global variable; let me see what else is doable without too much more time

Reply:
for expediency let's just use django settings https://docs.djangoproject.com/en/5.0/topics/settings/

You can set a new setting, e.g. `settings.SKIP_VIAF_LOOKUP`

And then you can check if it's true but default to false if it's not set (use `getattr`).

Backing up, though - are you importing VIAF ids for Person records that don't have birth and death date set? Don't we want those to be populated? Is there a bug in the logic that checks if the viaf lookup needs to run, when it's running when it shouldn't?

Reply:
No I'm only importing gender and nationality, I don't believe other fields were changed by our annotators.

Reply:
huh, interesting - did some quick investigation with my local data, I think we must have viaf set for a lot of authors where we don't have birth year and death year set - maybe because they aren't available in viaf.  (I was using `Person.objects.exclude(viaf_id="").filter(birth_year__isnull=True, death_year__isnull=True)` to look, but my data isn't very recent).

I wonder if the viaf logic is broken - some of these authors do seem like they should have birth/death years we can get from viaf, but the lookup isn't working.  I also see that we're hitting the viaf API twice instead of once. 🙄 

None of those are your problem here - bypassing for import still seems reasonable, but we might want to discuss with Josh - seems like author birth and death dates would be pretty useful information to have in the author demographics data we're going to be exporting. Maybe we can add a manage command to populate dates from viaf in bulk (and figure out why it isn't setting values properly)

Reply:
Did a little more digging - we have viaf IDs ending with slashes but the RDF has URIs ending _without_ the slash, so our lookups don't match and we don't get the dates we should be getting.  (Maybe something changed with VIAF since the time this was implemented, or maybe we changed our local practice.). We can fix that part in the S&co codebase.

I also see that the date parsing doesn't handle negative years... That needs to be fixed in viapy.